core/themes/basis/color/color.inc

```diff
@@ -26,15 +32,122 @@ $info = array(
         'base' => '#0074bd',
         'slogan' => '#000000',
         'titleslogan' => '#fffffe',
+        'hovermenu' => '#114a75',
+        'menutoggle' => '#bcbbbb',
+
         'bg' => '#ffffff',
-        'footer' => '#fffeff',
         'text' => '#000001',
         'link' => '#0073bd',
-        'hovermenu' => '#114a75',
-        'primarytabs' => '#586172',
         'borders' => '#bbbbbb',
         'formfocusborder' => '#43afe4',
+
+        'primarytabs' => '#586172',
+        'primarytabstext' => '#010000',
         'buttons' => '#dee2ea',
+
+        'footer' => '#fffeff',
+        'footertext' => '#000100',
+        'footerborder' => '#bbbbbc',
+      ),
+    ),
+    'ice' => array(
+      'title' => t('Ice'),
+      'colors' => array(
+        'header' => '#d0d0d0',
+        'base' => '#999999',
+        'slogan' => '#d0d0d1',
+        'titleslogan' => '#000000',
+        'hovermenu' => '#ffffff',
+        'menutoggle' => '#24272c',
+
+        'bg' => '#ffffff',
+        'text' => '#2a2a2a',
+        'link' => '#00809c',
+        'borders' => '#24272c',
+        'formfocusborder' => '#ffffff',
+
+        'primarytabs' => '#888889',
+        'primarytabstext' => '#d0d0d0',
+        'buttons' => '#014958',
+
+        'footer' => '#d0d0d2',
+        'footertext' => '#000100',
+        'footerborder' => '#24272c',
```

Comment:
The ice theme is generally rahter subtle, but the footer border is really dark. Can we change this to `#999999` (the same as `base`)

Reply:
looks great!